src/elements/Header/Header.js

```diff
@@ -159,4 +167,4 @@ Header.propTypes = {
 Header.Content = HeaderContent
 Header.Subheader = HeaderSubheader
 
-export default Header
+export default React.forwardRef(Header)
```

Comment:
Actually, this change should happen in the definition i.e. `const Header = React.forwardRef((props, ref) => {`

- we have statics (`Header.Content`, `Header.Subheader`) and it will not be usable on default export
- a param for `React.forwardRef()` is a function, not component itself:

https://github.com/facebook/react/blob/41694201988c5e651f0c3bc69921d5c9717be88b/packages/react/src/ReactForwardRef.js#L38-L41